lib/shopify_transporter/pipeline/magento/order/shipping_lines.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+require 'shopify_transporter/pipeline/stage'
+require 'shopify_transporter/shopify'
+
+module ShopifyTransporter
+  module Pipeline
+    module Magento
+      module Order
+        class ShippingLines < Pipeline::Stage
+          def convert(input, record)
+            record.merge!(
+              {
+                shipping_lines: shipping_lines(input),
+              }.stringify_keys
+            )
+          end
+
+          private
+
+          def shipping_lines(input)
+            [
+              {
+                code: input['shipping_description'],
+                title: input['shipping_description'],
+                price: input['shipping_amount'],
+                carrier_identifier: input['shipping_method'],
+                tax_lines: shipping_tax_lines(input),
+              }.stringify_keys,
+            ]
+          end
+
+          def shipping_tax_lines(input)
+            return unless input['shipping_tax_amount'].present? && input['shipping_tax_amount'].to_f != 0
+
+            [
+              {
+                price: input['shipping_tax_amount'],
```

Comment:
Why not just `'price': input['shipping_tax_amount']` instead and avoid the call to `stringify_keys`? I'd probably make this change throughout this file just because it seems a little roundabout if we know we expect string keys but use symbols and then only to convert them.

Reply:
Make sense! I did the call of `stringify_keys` just because most of pipeline stages are using it and I followed the same pattern. I'll update the code.

Reply:
`'price': input['shipping_tax_amount']` is equivalent to `price: input['shipping_tax_amount']` as it's just syntactic sugar to be able to specify a string with a colon (but ruby still sets it to a symbol).

Forget this comment, I forgot about that ^